samples/subsys/input/input_dump/src/main.c

```diff
@@ -6,8 +6,50 @@
 
 #include <stdio.h>
 
+#ifdef CONFIG_SHELL_BACKEND_DUMMY
+#include <zephyr/shell/shell.h>
+#include <zephyr/shell/shell_dummy.h>
+
+#define SLEEP_TIME_MS 200
+#define CMD_BUF_LEN 128
+
+static void test_input_shell_command(void)
+{
```

Comment:
hmm isn't this a bit confusing in `samples/`? shouldn't this be in `tests/`?

Reply:
Yeah it does look messy. Let me do some copy paste and move that in a dedicated test.

Reply:
Sorted.